Dockerfile

```diff
@@ -36,7 +36,7 @@ COPY $DOCS_DIR/pom.xml ./$DOCS_DIR/
 COPY $OPERATOR_DIR/src ./$OPERATOR_DIR/src
 COPY $WEBHOOK_DIR/src ./$WEBHOOK_DIR/src
 
-COPY .git ./.git
+COPY *.git ./.git
```

Comment:
I'm unsure whether this line can possibly pollute unexpected files. At least we should add a comment about this tricky - it's a workaround since Dockerfile doesn't support conditional copy directly.

Reply:
@tisonkun, I have removed the `.git` directory and built image successfully. The change will copy the `.git` directory when the directory exists and not copy this directory when the `.git` directory doesn't exist.